modules/dnn/test/test_common.hpp

```diff
@@ -137,7 +136,6 @@ bool validateVPUType();
 
 testing::internal::ParamGenerator< tuple<Backend, Target> > dnnBackendsAndTargets(
         bool withInferenceEngine = true,
-        bool withHalide = false,
```

Comment:
Not sure that this is safe to remove intermediate default argument. Maybe it's better to keep for now and add `CV_Assert(!withHalide)` inside?